src/commands/createApp.ts

```diff
@@ -54,10 +55,14 @@ export async function createApp(
 
     spinner.succeed('Created new Belt app with Expo');
 
+    const packageManager = getPackageManager(options);
+    if (packageManager === 'npm') {
+      await addNpmrc(appName);
+    }
+
```

Comment:
This is very minor but do you think it would it be safer to move this part to after the `process.chdir(`./${appName}`);`? That way if anything goes wrong with the `appName` it will fallback to the project itself.